docs/sdk-and-tools/indices/tokens.md

```diff
@@ -0,0 +1,35 @@
+---
+id: es-index-tokens
+title: Elasticsearch index: tokens
+---
+
+The `_id` field of this index is represented by token identifier of an ESDT token.
+
+| Field         | Description                                                                                                                 |
+|---------------|-----------------------------------------------------------------------------------------------------------------------------|
+| name          | The name field holds the name of the token. It contains alphanumeric characters only.                                       |
+| ticker        | The ticker field represents the unique combination of letters assigned to a token.                                          |
```

Comment:
Wrong. The ticker is not unique and it can contain numbers as well.

Update to:

The ticker field represents the token's ticker (uppercase alphanumeric characters).

Reply:
changed.